index.js

```diff
@@ -1,36 +1,35 @@
 module.exports = (nextConfig = {}) => {
   return Object.assign({}, nextConfig, {
     webpack(config, options) {
-      const { isServer } = options;
+      const { isServer } = options
 
       if (!options.defaultLoaders) {
         throw new Error(
-          'This plugin is not compatible with Next.js versions below 5.0.0 https://err.sh/next-plugins/upgrade'
+          "This plugin is not compatible with Next.js versions below 5.0.0 https://err.sh/next-plugins/upgrade"
         )
       }
 
-      const prefix = nextConfig.assetPrefix || '';
-      const directory = nextConfig.assetDirectory || 'public';
+      const prefix = nextConfig.assetPrefix || ""
 
       config.module.rules.push({
         test: /\.(mp4|webm|ogg|swf|ogv)$/,
         use: [
           {
-            loader: require.resolve('file-loader'),
+            loader: require.resolve("file-loader"),
             options: {
-              publicPath: `${prefix}/_next/${directory}/videos/`,
-              outputPath: `${isServer ? '../' : ''}${directory}/videos/`,
-              name: '[name]-[hash].[ext]',
+              publicPath: `${prefix}/_next/static/videos/`,
```

Comment:
Alright, I might have misunderstood the changelog at that time. What about the previous version of next.js prior to 9.1, will it also works with `static` directory?

Another possibility, if we are not sure how it works on all version of next, to keep the config and default it to static.

Reply:
I can spin up an older version of Nest app soon, tonight or tomorrow, and do some testing without the config using just the `/static` path. Ideally we wouldn't keep anything in the API that isn't necessary.

Reply:
@jeremybarbet Just tested using Next.js version `8.1.0` and using `/static` as the path works.